.github/workflows/build.yaml

```diff
@@ -106,17 +102,13 @@ jobs:
     needs:
       - "rustfmt"
       - "markdown-lint"
-    strategy:
-      matrix:
-        rust:
-          - nightly-2022-07-22
     steps:
       - uses: actions/checkout@v3
       - uses: mobilecoinfoundation/actions/dcap-libs@main
       - uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: ${{ matrix.rust }}
+          toolchain: stable
```

Comment:
For both `clippy` and `rustfmt`, we should actually fail on `stable`, but generate warnings for `beta`---the most likely thing to cause a CI fail on a stable upgrade is going to be `clippy` adding a lint.

Doesn't need to happen in this PR, but it should in a follow-up.

Reply:
Created #187